js/components/frame.js

```diff
@@ -54,12 +54,6 @@ class Frame extends ImmutableComponent {
 
   componentDidMount () {
     this.updateWebview()
-    // forward postMessage events from webview to webContents
-    window.addEventListener('message', function (event) {
-      if (this.webview.getAttribute('src').match(event.origin)) {
```

Comment:
Yep per slack match origin will also match things like brianbondy.com#file:// and allow it. So this should be startsWith I think and also I'm not sure how to lock this to a full path and not just the file protocol.